src/models/AccessLog.ts

```diff
@@ -29,10 +29,10 @@ export class AccessLog {
   accessDate: Date;
 
   @Column({
-    type: 'enum',
+    type: 'enum', 
     enum: ActionType,
   })
-  action: ActionType;
+    action: ActionType;
```

Comment:
Please remove extra spaces (yeah it seems that the issue with eslint still persists)